### ISSUE TITLE
scale_range description

### ISSUE THREAD

**Issue by k-dominik** [bioimage-io/spec-bioimage-io#73]
More a question. I'm trying to understand the spec by reading it. The `scale_range` description seems to require some domain knowledge to understand. I found a few descriptions on "percentile normalization" (e.g. [one an implementation here](https://github.com/seangibbons/percentile_normalization)). Would it be possible to add a link to where this transformation is described, or describe it in the spec?

**Comment by oeway** [comment]
I agree, we should definitely describe these, and provide reference implementation, e.g. in our python library.

**Comment by k-dominik** [comment]
We have a good reference implementation now in core, so I'm good :)
